horaedb/metric_engine/src/sst.rs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use crate::types::ObjectStoreRef;
+
+pub struct SSTable {
+    pub sst_id: u64,
+    pub storage: ObjectStoreRef,
+}
```

Comment:
It seems storage should not be part of SSTable? The struct seems better to include the metadata(e.g. id, path)
Maybe we just define a empty struct in this sketch?

Reply:
OK, I will just leave an `id` in it.